docs/usage.md

```diff
@@ -50,6 +50,8 @@ env:
   TFBUDDY_PROJECT_ALLOW_LIST: tfc-project/
   TFBUDDY_WORKSPACE_ALLOW_LIST: tfc-workspace
   TFBUDDY_DEFAULT_TFC_ORGANIZATION: companyX
+  # Optional setting to disable auto merging MRs after a successful apply. This is enabled by default.
+  TFBUDDY_ENABLE_AUTO_MERGE: "false"
```

Comment:
sorry, one last thing. since this doesn't enable it, but rather prevents it from being enabled per repo, can we rename this "ALLOW_AUTO_MERGE", or something similar?